base/main_test_bucket_pool.go

```diff
@@ -318,6 +322,44 @@ func (tbp *TestBucketPool) GetWalrusTestBucket(t testing.TB, url string) (b Buck
 	}
 }
 
+// GetExistingBucket opens a bucket conection to an existing bucket
+func (tbp *TestBucketPool) GetExistingBucket(t testing.TB) (b Bucket, s BucketSpec, teardown func()) {
+	testCtx := TestCtx(t)
+
+	bucketCluster := initV2Cluster(UnitTestUrl())
+
+	var useNamedCollections bool
+	useDefaultCollectionString, isSet := os.LookupEnv(tbpUseDefaultCollection)
+	if isSet {
+		useDefaultCollection, _ := strconv.ParseBool(useDefaultCollectionString)
+		useNamedCollections = !useDefaultCollection
+	} else {
+		if !TestsDisableGSI() {
+			useNamedCollections = true
+		}
+	}
+
+	bucketName := tbpBucketName(TestUseExistingBucketName())
+	bucketSpec := getBucketSpec(bucketName, useNamedCollections)
+
+	bucket := bucketCluster.Bucket(bucketSpec.BucketName)
+	err := bucket.WaitUntilReady(10*time.Second, nil)
```

Comment:
Can we turn this into a test constant https://cs.github.com/couchbase/sync_gateway/blob/d1ce9ca698e6e3a4474e74886fb41a5bd4648e6a/base/main_test_cluster.go?q=waituntilready#L149 ? I notice different times on server 6.6 and I'd rather modify in one spot.

Reply:
I think I'll just set everything to use the `waitForReadyBucketTimeout = time.Minute` const - will take slightly longer in the failure case but happy-path shouldn't be affected by the increased timeout.